pandas/tseries/tests/test_tslib.py

```diff
@@ -713,11 +718,17 @@ def test_parsers_iso8601(self):
             self.assertEqual(actual, exp)
 
         # seperators must all match - YYYYMM not valid
-        invalid_cases = ['2011-01/02', '2011^11^11', '201401',
-                         '201111', '200101']
+        invalid_cases = ['2011-01/02', '2011^11^11',
+                         '201401', '201111', '200101',
+                         '2005-0101', '200501-01',
+                         '20010101 1', '20010101 123', '20010101 12345',
+                         '20010101 12:3456', '20010101 1234:56',
```

Comment:
can you add some commentary here of why some of these are invalid (e.g. the not-so-obvious ones), you can do it inline with a comment for those that are appropriate